src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,53 @@
 package core.basesyntax.service;
 
+import core.basesyntax.DataNotVaidExeption;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int LOGIN_MIN_LENGHT = 6;
+    private static final int USER_PWD_LENGHT = 6;
+    private static final int USER_MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new DataNotVaidExeption("User can`t be NULL");
+        }
+        if (user.getPassword() == null || user.getLogin() == null) {
+            throw new DataNotVaidExeption("Login and password can`t be null");
+        }
+        if (Storage.people.size() > 0) {
```

Comment:
if size is 0 for loop just won't iterate, but here we don't need for loop anyway